Source/WebCore/Modules/model-element/HTMLModelElement.cpp

```diff
@@ -72,9 +73,10 @@ using namespace HTMLNames;
 WTF_MAKE_ISO_ALLOCATED_IMPL(HTMLModelElement);
 
 HTMLModelElement::HTMLModelElement(const QualifiedName& tagName, Document& document)
-    : HTMLElement(tagName, document, { TypeFlag::HasCustomStyleResolveCallbacks, TypeFlag::HasDidMoveToNewDocument })
```

Comment:
This style change is wrong

Reply:
somehow the formatter wants me to do that 🤔 said "things inside a namespace should not be indented"
I guess I'll just keep it as-is and ignore the formatter